src/vorta/views/repo_tab.py

```diff
@@ -220,10 +220,7 @@ def add_existing_repo(self):
         # window.rejected.connect(lambda: self.repoSelector.setCurrentIndex(0))
         window.open()
 
-    def repo_select_action(self, index):
-        if index == 0:
-            return
-
+    def repo_select_action(self):
```

Comment:
```suggestion
def repo_select_action(self, index: int):
```

The method is connected to the `currentIndexChanged(int index)` signal of the QComboBox `repoSelector`.

Reply:
For some reason it still works and doesn't produce an error. :shrug:

Reply:
Probably Qt function overloading. I tested it of course.